app/lib/pdf/finance/invoice_generator.rb

```diff
@@ -102,6 +102,7 @@ def print_details
       end
 
       def print_line_items
+        @pdf.text "Line items"
```

Comment:
use `subtitle` like above?

Reply:
No need:
<img width="978" alt="Screenshot 2021-02-10 at 16 45 09" src="https://user-images.githubusercontent.com/13486237/107533662-732a2700-6bbf-11eb-9428-08065ae23e85.png">

Reply:
Visually not but if you would parse the pdf with a screenreader it would be better as it adds semantics. And if one we decide to style sub headers a certain way too.